rollup-interface/src/state_machine/da.rs

```diff
@@ -64,4 +66,6 @@ pub trait BlobTransactionTrait: Serialize + DeserializeOwned {
     fn data(&self) -> Self::Data;
 }
 
+/// TODO: This trait has nothing specific to BlockHash, it just set of bounds
```

Comment:
I think we want to keep a specific `BlockHashTrait` so that we can easily add methods which will only apply to block hashes